interactions.go

```diff
@@ -227,12 +227,6 @@ type InteractionResponseType uint8
 const (
 	// InteractionResponsePong is for ACK ping event.
 	InteractionResponsePong = InteractionResponseType(iota + 1)
-	// InteractionResponseAcknowledge is for ACK a command without sending a message, eating the user's input.
-	// NOTE: this type is being imminently deprecated, and **will be removed when this occurs.**
-	InteractionResponseAcknowledge
-	// InteractionResponseChannelMessage is for responding with a message, eating the user's input.
-	// NOTE: this type is being imminently deprecated, and **will be removed when this occurs.**
-	InteractionResponseChannelMessage
```

Comment:
Removing these outright will mess up the `iota`
Consider replacing with `_` and comments if needed

Reply:
is using the iota even worth it for this anymore. could just actually write `1, 4, 5`?

Reply:
It sounds like the consensus in Discord is to hard-code them. 😊